### ISSUE TITLE
[COMPAT] Crash when viewing echo boat with WTHIT

### ISSUE THREAD

**Issue by lHooded** [KyaniteMods/DeeperAndDarker#112]
**Mods**
Deeper and Darker 1.1.4
Geckolib 3.1.37
Fabric API 0.68.0
WTHIT 5.13.4
Bad Packets 0.2.1

**Mod Loader**
Fabric 0.14.11 (Have not tested Forge)

**Description**:
When I place an echo boat with WTHIT installed, the game crashes. This does not happen with vanilla boats / other modded boats.

**Steps to reproduce the behavior**:
1. Load mods as shown
2. Create new world
3. Place and view echo boat

**Comment by NewJumper** [comment]
what is WTHIT?

**Comment by lHooded** [comment]
Additionally, when I view the echo boat with 8.6.3 JADE (www.curseforge.com/minecraft/mc-mods/jade) on 1.19.2, the boat does not appear in Jade's added HUD.

**Comment by TigerWalts** [comment]
Here's the stack trace from the client crash:

```
java.lang.NullPointerException: Cannot invoke "java.util.function.Supplier.get()" because "this.item" is null
	at com.kyanite.deeperdarker.registry.entities.custom.DDBoat$Type.getItem(DDBoat.java:113)
	at com.kyanite.deeperdarker.registry.entities.custom.DDBoat.method_7557(DDBoat.java:66)
	at net.minecraft.class_1690.method_31480(class_1690.java:995)
	at mcp.mobius.waila.plugin.core.provider.EntityProvider.getIcon(EntityProvider.java:32)
	at mcp.mobius.waila.gui.hud.ComponentHandler.getIcon(ComponentHandler.java:114)
	at mcp.mobius.waila.gui.hud.TooltipHandler.tick(TooltipHandler.java:186)
	at mcp.mobius.waila.WailaClient.onClientTick(WailaClient.java:63)
	at mcp.mobius.waila.fabric.FabricWailaClient.lambda$onInitializeClient$0(FabricWailaClient.java:24)
	at net.fabricmc.fabric.api.client.event.lifecycle.v1.ClientTickEvents.lambda$static$2(ClientTickEvents.java:65)
	at net.minecraft.class_310.handler$eja000$fabric-lifecycle-events-v1$onEndTick(class_310.java:17980)
	at net.minecraft.class_310.method_1574(class_310.java:1907)
	at net.minecraft.class_310.method_1523(class_310.java:1135)
	at net.minecraft.class_310.method_1514(class_310.java:768)
	at net.minecraft.client.main.Main.method_44604(Main.java:244)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:468)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
```

**Comment by Pedro270707** [comment]
Fixed in 1.2.0
